### PR TITLE
feat: Add night restrictions based on offer responses

### PR DESCRIPTION
**Pull request by jorelosorio**
Closes https://github.com/AtB-AS/kundevendt/issues/2837
Depends on: https://github.com/AtB-AS/firestore-configuration/pull/41

### CODE REVIEW

File: src/translations/screens/subscreens/PurchaseOverview.ts [hunk 1]
```diff
@@ -8,6 +8,11 @@ const PurchaseOverviewTexts = {
       'Oops - vi klarte ikke å søke opp prisen. Supert om du prøver igjen 🤞',
       'Whoops - we were unable to retrieve cost. Please try again 🤞',
     ),
+    producUnavailable: (productName: string) =>
+      _(
+        `Produktet ${productName} er kanskje ikke tilgjengelig akkurat nå.`,
+        `Product ${productName} might not be available right now.`,
+      ),
```
Comment (jorelosorio) [inline src/translations/screens/subscreens/PurchaseOverview.ts:15]: @ckbilstad What do you think about these texts? ;)

**Review by gorandalum — COMMENTED** [review]
After team talk and talk with Christian: Ok to just have the time limitation message as part of the sub text with a line break.

Screenshot 2023-01-16 at 15 51 15

**Comment by jorelosorio** [comment]
> After team talk and talk with Christian: Ok to just have the time limitation message as part of the sub text with a line break.
> 
> Screenshot 2023-01-16 at 15 51 15

Changes were made after discussion. However, there is a behavior that changed based on the original thought:

- The warning/restriction message is hidden when the user toggle `Vis info` @gorandalum agreed on is can be dismissed together with the product description.

NOTE: @ckbilstad Please take a look to the texts if they looks good.

**Review by gorandalum — APPROVED** [review]

File: src/translations/screens/subscreens/PurchaseOverview.ts (hunk 1 above)
Comment (jorelosorio) [reply src/translations/screens/subscreens/PurchaseOverview.ts:15]: I had a talk with Christian and the texts were changed as the image below:

Screenshot_1673947628

**Pull request merged**